docs/usage.md

```diff
@@ -1,21 +1,32 @@
 # Usage
 
-## Component Project
+The [C#/WinRT NuGet package](https://www.nuget.org/packages/Microsoft.Windows.CsWinRT/) provides tooling for the following scenarios:
 
-A component project adds a NuGet reference to C#/WinRT to invoke cswinrt.exe at build time, generate projection sources, and compile these into an interop assembly. For an example of this, see the [Projection Sample](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/Net5ProjectionSample). Command line options can be displayed by running **cswinrt -?**.  The interop assembly is then typically distributed as a NuGet package itself. 
+- [Generate and distribute an interop assembly](#generate-and-distribute-an-interop-assembly)
+- [Author and consume a C#/WinRT component](#author-and-consume-a-cwinrt-component)
 
-### Example 
+For more information on using the NuGet package, refer to the [NuGet documentation](../nuget/readme.md).
 
-To invoke cswinrt to generate projection sources for types in the **Contoso** namespace, you need to add the following property to your C# library project file. In this project you would also need to reference the CsWinRT NuGet package and the project-specific `winmd` files you want to project, whether through a NuGet package, project reference or direct reference. By default, the **Windows** and **Microsoft** namespaces are not projected.
+## Generate and distribute an interop assembly
+
+A projection project adds a NuGet reference to C#/WinRT to invoke cswinrt.exe at build time, generate projection sources, and compile these into an interop assembly. For an example of this, see the [Projection Sample](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/Net5ProjectionSample) and associated [walkthrough](https://docs.microsoft.com/windows/uwp/csharp-winrt/net-projection-from-cppwinrt-component). Command line options can be displayed by running `cswinrt -?`.  The interop assembly is then typically distributed as a NuGet package itself for application projects to use.
+
+### Generate Projections
```

Comment:
If I recall correctly, CsWinRTIncludes is needed for a projection to be generated right?  Due to that, should we merge both sections and without CsWinRTIncludes, you won't build an interop assembly.

Reply:
Not sure I understand, should we not show the `CswinRTIncludes` property and just let folks refer to the projection sample for if they want to generate their own projection?

Reply:
No, we should include it.  I was just saying the flow seemed a bit confusing where we were talking about generating interop assemblies and then generating projections when both are connected to each other.  I feel like both together should be a section where we talk about we generate projection interop assemblies based on the types listed as to be projected in CsWinRTIncludes.

Reply:
do these changes look ok

Reply:
Looks good.